nuprojs/Microsoft.ServiceFabric.Actors/Microsoft.ServiceFabric.Actors.nuproj

```diff
@@ -148,12 +148,18 @@
       </File>
       <File Include="$(DropFolderNetStandard_Win)system.fabric.strings.dll">
         <TargetPath>build\netcoreapp2.0</TargetPath>
+      </File>
+	  <File Include="$(DropFolderNetStandard_Win)Microsoft.ServiceFabric.StartupServicesUtility.dll">
+        <TargetPath>build\netcoreapp2.0</TargetPath>
       </File>
     </ItemGroup>
     
     <ItemGroup>
       <Dependency Include="Microsoft.ServiceFabric.Services.Remoting">
         <Version>[$(NuGetPackageVersion)]</Version>
       </Dependency>
+      <Dependency Include="Microsoft.ServiceFabric.StartupServicesUtility">
+        <Version>[$(NuGetPackageVersion)]</Version>
```

Comment:
Do I need to add this dependency in Microsoft.ServiceFabric.Actors.Wcf nuproj as well ?
I believe no as wfc has given dependency on Actors, hence it is covered.